src/Nri/Ui/Modal/V5.elm

```diff
@@ -238,18 +267,28 @@ viewFooter =
 
 
 {-| -}
-launchButton : (Msg -> msg) -> List Css.Style -> String -> Html msg
-launchButton wrapMsg styles label =
-    button
-        (css styles
-            :: List.map Html.Styled.Attributes.fromUnstyled
-                (Modal.openOnClick wrapMsg (String.replace " " "-" label))
-        )
-        [ text label ]
+launchButton : (Msg -> msg) -> String -> Html msg
```

Comment:
❓ What is a launch button?  Is this for opening the modal?  Wouldn't different events cause different modals to open?  I'm confused about what this is for... if you know, could you explain it in the doc comments for this function?